packages/@react-spectrum/accordion/stories/Accordion.stories.tsx

```diff
@@ -20,53 +20,55 @@ type ItemType = {
   title: string
 };
 
-/**
- * Helper type so `bind` returns the actual Story type.
- */
-interface Story<T> extends _Story<T> {
-  bind: (this: ThisParameterType<typeof Function.bind>, thisArg: Parameters<typeof Function.bind>[0], ...argArray: Parameters<typeof Function.bind>[1][]) => _Story<T>
-}
-
 export default {
   title: 'Accordion',
-  component: Accordion
-} as Meta<SpectrumAccordionProps<ItemType>>;
+  component: Accordion,
+  argTypes: {}
+} as ComponentMeta<typeof Accordion>;
+
+export type AccordionStory = ComponentStoryObj<typeof Accordion>;
+
+export const Default: AccordionStory = {
+  args: {
+    items: [
+      {key: 'files', title: 'Your files'},
+      {key: 'shared', title: 'Shared with you'},
+      {key: 'last', title: 'Last item'}
+    ]
+  },
+  render: (args) => (
+    <Accordion {...args}>
+      {(item) => <Item key={(item as ItemType).key} title={(item as ItemType).title}>{(item as ItemType).key}</Item>}
+    </Accordion>
+  )
+};
+
+export const DefaultExpandedKeys: AccordionStory = {
```

Comment:
default-expanded-keys, disabled-keys, and disabled-default-expanded-keys are not rendering in storybook.

Reply:
ah, i think i see, i wasn't providing a render function. don't know why that worked locally